python/setup.py

```diff
@@ -209,11 +207,22 @@ def get_win_arch():
         'Intended Audience :: Developers',
         'Intended Audience :: Science/Research',
         'License :: OSI Approved :: Apache Software License',
-        'Operating System :: Unix',
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: Microsoft :: Windows',
+        'Operating System :: POSIX :: Linux',
```

Comment:
While there is nothing that _shouldn't_ work on other platforms, there are no wheels for them and they are untested.